web-console/5-components/order-from-catalog.md

```diff
@@ -50,58 +40,68 @@ The button panel includes navigation buttons: Cancel, Back and Next/Create/Close
 - The Next button should become enabled once all required information has been entered for the current step and/or sub-step. The Next button will move the user through any sub-steps before it moves the user to the next main step. The Back button will also behave the same way.  
 
 ## Possible Steps
-There are 3 possible steps in this process: Plan, Configuration and Results.
-
-Post 3.6, an additional Bind Step will be implemented.
-
-### Step 1 - Plan
-
-### User selects a plan
-
-#### Catalog item has a single version
+The following steps may be part of the this process:
+1. Information
+1. Plan (optional)
+1. Configuration
+1. Bind (optional)
+1. Parameters (optional)
+1. Results
+
+### 1 - Information Step
+
+![template](img/information-step.png)
+
+1. Catalog Item Icon
+	- metadata.imageURL from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Catalog Item Name
+	- metadata.displayName from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Provider (Optional)
+	- metadata.providerDisplayName from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. List of tags (Optional)
+1. Short description (Optional)
+1. Long description (Optional)
+	- metadata.longDescription from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+1. Image dependencies : Includes an **Image Dependencies** title following by a list of dependencies.  Each image name is preceded by an image icon (note the PF icon)
+1. **View documentation** link which will open a new browser/tab linking to the Documentation URL (Optional)
+	- metadata.documentationUrl from [OpenServiceBroker Service Metadata](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#service-metadata-fields)
+
+### 2 - Plan Step (Optional)
+This step should be shown if there are 1+ plans.
+
+#### Single Plan
 ![template](img/plan-step.png)
```

Comment:
I think this image and the next one should be flip-flopped.

Reply:
Agreed @ncameronbritt good catch!